### ISSUE TITLE
test scenario: invalid wml

### ISSUE THREAD

**Issue by matthiaskrgr** [wesnoth/wesnoth#1720]
````
./build/wesnoth -t
````
````
Battle for Wesnoth v1.13.8+dev (ae111a4-Modified)
Started on Thu May 25 13:41:23 2017

Automatically found a possible data directory at /home/matthias/vcs/github/wesnoth/build/..

Data directory:               /home/matthias/vcs/github/wesnoth/build/..
User configuration directory: /home/matthias/.config/wesnoth
User data directory:          /home/matthias/.local/share/wesnoth/1.13
Cache directory:              /home/matthias/.cache/wesnoth

Setting mode to 1026x861
Checking lua scripts... ok
20170525 13:41:28 warning wml: helper.set_wml_tag_metatable is deprecated; use wml.tag instead which has the metatable already set
20170525 13:41:32 warning wml: helper.shallow_literal is deprecated; use wml.shallow_literal instead
20170525 13:41:32 warning wml: helper.get_child is deprecated; use wml.get_child instead
20170525 13:41:32 warning wml: wesnoth.set_variable is deprecated; use wml.variable.set instead
20170525 13:41:32 warning wml: wesnoth.get_variable is deprecated; use wml.variable.get instead
20170525 13:41:32 warning wml: helper.get_variable_array is deprecated; use wml.variable.get_array instead
20170525 13:41:32 warning wml: helper.shallow_parsed is deprecated; use wml.shallow_parsed instead
20170525 13:41:32 warning wml: helper.parsed is deprecated; use wml.parsed instead
20170525 13:41:32 warning wml: helper.child_count is deprecated; use wml.child_count instead
20170525 13:41:32 warning wml: helper.child_range is deprecated; use wml.child_range instead
20170525 13:41:32 warning wml: helper.get_variable_proxy_array is deprecated; use wml.variable.get_proxy_array instead
20170525 13:41:33 warning wml: wesnoth.tovconfig is deprecated; use wml.tovconfig instead
20170525 13:41:33 warning wml: helper.literal is deprecated; use wml.literal instead
Invalid WML found: [unstore_unit]: variable 'LUA_modify_unit[0]' doesn't contain unit data
````

**Comment by gfgtdf** [comment]
hmm the last erro message comes form my last commit. will look at it

**Comment by gfgtdf** [comment]
fixed by ea5dc137ed3ad4f6fb9d0f2a8747917922d69a01
